ext/oci8/oci8.stub.php

```diff
@@ -951,8 +951,10 @@ function oci_register_taf_callback($connection, ?callable $callback): bool {}
 /** @param resource $connection */
 function oci_unregister_taf_callback($connection): bool {}
 
-#[\AllowDynamicProperties]
 class OCILob {
+    /** @var resource */
+    public mixed $descriptor = null;
```

Comment:
as far as I managed to understand, the property is private indeed as you guessed... Would be nice to double check with someone else and change the visibility accordingly. If someone later complains, we can still make it `public` again.

Reply:
Yep that's why I asked @cjbj for a confirmation. I'm not an OCI user and looking at some tests I see stuff like `unset($clob->descriptor);`.

Reply:
I won't have time to look at this until next month :(